<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapBrowser</class>
 <widget class="QMainWindow" name="MapBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map Explorer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/mapbrowser.ico</normaloff>:/images/mapbrowser.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="MapTree" name="treeWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>300</height>
       </size>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="MapWebView" name="webView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>483</width>
     <height>20</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuIdioma">
    <property name="title">
     <string>Configurações</string>
    </property>
    <widget class="QMenu" name="menuIdioma_2">
     <property name="title">
      <string>Idioma</string>
     </property>
     <addaction name="actionPortugu_s"/>
     <addaction name="actionEnglish"/>
    </widget>
    <addaction name="actionAlterar_a_pasta_raiz"/>
    <addaction name="separator"/>
    <addaction name="menuIdioma_2"/>
    <addaction name="actionEsconder_a_lista_de_pastas"/>
    <addaction name="actionUsar_o_tema_CleanLooks"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Sobre</string>
    </property>
    <addaction name="actionSobre"/>
    <addaction name="separator"/>
    <addaction name="actionSobre_Qt"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuIdioma"/>
   <addaction name="menuSobre"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSobre_Qt">
   <property name="text">
    <string>Sobre Qt</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/about.ico</normaloff>:/images/about.ico</iconset>
   </property>
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionPortugu_s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Português</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionUsar_o_tema_CleanLooks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Usar o tema CleanLooks</string>
   </property>
  </action>
  <action name="actionAlterar_a_pasta_raiz">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/folder.ico</normaloff>:/images/folder.ico</iconset>
   </property>
   <property name="text">
    <string>Alterar a pasta raiz</string>
   </property>
  </action>
  <action name="actionEsconder_a_lista_de_pastas">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Esconder a lista de mapas</string>
   </property>
   <property name="toolTip">
    <string>Esconder a lista de mapas</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>MapTree</class>
   <extends>QTreeWidget</extends>
   <header>maptree.h</header>
  </customwidget>
  <customwidget>
   <class>MapWebView</class>
   <extends>QWebView</extends>
   <header>mapwebview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionEsconder_a_lista_de_pastas</sender>
   <signal>toggled(bool)</signal>
   <receiver>treeWidget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEsconder_a_lista_de_pastas</sender>
   <signal>toggled(bool)</signal>
   <receiver>line</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
